ttyclock.c

```diff
@@ -569,10 +569,11 @@ main(int argc, char **argv)
                       "    -b            Use bold colors                                \n"
                       "    -t            Set the hour in 12h format                     \n"
                       "    -u            Use UTC time                                   \n"
-              "    -T tty        Display the clock on the specified terminal    \n"
+                      "    -z [tz]       Use the specified timezone                     \n"
```

Comment:
i'm confused: i closed #50 saying:

> this is what the TZ environment variable is for. i don't think we should implement an option where there is already a standard environment variable knob.

why should we add an option then? :) the improvement on top of that might have been to mention it in the manpage, but i wouldn't add a commandline option... there are already a *lot* of options for a simple clock, so I'm very hesitant in accepting more...

sorry about that, and thanks for your work!

Reply:
I just think the option is neat. This:

```bash
tmux new -d -s test; tmux splitw -h; \
tmux splitw -v; tmux splitw -v; tmux selectp -t 1; tmux splitw -v; tmux selectp -t 0; \
tmux splitw -v; tmux splitw -v; tmux selectp -t 0; tmux splitw -v; \
tmux send-keys -t 0 "env TZ=America/Los_Angeles ./tty-clock -c" Enter; \
tmux send-keys -t 1 "env TZ=Asia/Tel_Aviv       ./tty-clock -c" Enter; \
tmux send-keys -t 2 "env TZ=America/Los_Angeles ./tty-clock -c" Enter; \
tmux send-keys -t 3 "env TZ=Asia/Singapore      ./tty-clock -c" Enter; \
tmux send-keys -t 4 "env TZ=America/New_York    ./tty-clock -c" Enter; \
tmux send-keys -t 5 "env TZ=Europe/Paris        ./tty-clock -c" Enter; \
tmux send-keys -t 6 "env TZ=Europe/Moscow       ./tty-clock -c" Enter; \
tmux send-keys -t 7 "env TZ=Australia/Canberra  ./tty-clock -c" Enter; \
tmux attach -t test
```
 doesnt look as good as this:

```bash
tmux new -d -s test; tmux splitw -h; \
tmux splitw -v; tmux splitw -v; tmux selectp -t 1; tmux splitw -v; tmux selectp -t 0; \
tmux splitw -v; tmux splitw -v; tmux selectp -t 0; tmux splitw -v; \
tmux send-keys -t 0 "./tty-clock -c -z America/Los_Angeles" Enter; \
tmux send-keys -t 1 "./tty-clock -c -z Europe/London" Enter; \
tmux send-keys -t 2 "./tty-clock -c -z Asia/Tel_Aviv" Enter; \
tmux send-keys -t 3 "./tty-clock -c -z Asia/Singapore" Enter; \
tmux send-keys -t 4 "./tty-clock -c -z America/New_York" Enter; \
tmux send-keys -t 5 "./tty-clock -c -z Europe/Paris" Enter; \
tmux send-keys -t 6 "./tty-clock -c -z Europe/Moscow" Enter; \
tmux send-keys -t 7 "./tty-clock -c -z Australia/Canberra" Enter; \
tmux attach -t test
```
In my opinion it's a tradeoff between a slightly smaller codebase and a slightly better usability.

Reply:
i'm sorry, but i won't accept this patch s is, as i said in #50. all i will accept will be documentation fixes to tech people about TZ... :/

for the record, you don't need `env`, most shells can pass environment with:

```
VAR=val command -h
```

for example, your tmux config would become:

```
tmux send-keys -t 0 "TZ=America/Los_Angeles ./tty-clock -c" Enter; \
```

`TZ=` is nnly two *one* character more than `-z` and, if you count the space, it's actually the same number of characters. :) It's true that `TZ` is less *discoverable* though, which is why i agree it's also less usable. But I don't believe the fix for that is to add a new flag. Instead we should document that the program obeys `TZ` as it should.

I'll close this PR now, I would be happy to take a patch that improves the documentation instead.